project-base/src/DataFixtures/Demo/ProductDataFixture.php

```diff
@@ -72,26 +73,26 @@ class ProductDataFixture extends AbstractReferenceFixture implements DependentFi
      */
     private $parameterDataFactory;
 
-    /**
-     * @var \Shopsys\FrameworkBundle\Model\Product\Parameter\Parameter[]
-     */
-    private $parameters;
-
```

Comment:
Why have been parameters removed here?

Reply:
I guess the main reason is due to persistence reference.